assets/scripts/TaroCard.js

```diff
@@ -0,0 +1,63 @@
+class TarotCard{
+    constructor(){
+
+    }
+    /**
+     * Moves object from one point to another over a duration of time
+     *
+     * @param pointA The object's starting point
+     * @param pointB The object's ending point
+     * @param time Time in milliseconds it takes for the object to move from
+     * pointA to pointB
+     */
+    move(pointA, pointB, time) {
+        //
+    }
+
+
+    /**
+     * Moves object from one point to another instantly
+     *
+     * @param pointA The object's starting point
+     * @param pointB The object's ending point
+     */
+    moveInstantly(pointA, pointB) {
+        //
+    }
+
+    /**
+     * Rotates object some number of degrees over a duration of time
+     *
+     * @param degrees Number of degrees to rotate object
+     * @param time Time in milliseconds it takes for the object to complete the 
+     * rotation
+     */
+    rotate(degrees, time) {
+        // 
+    }
+
+    /**
+     * Rotates object some number of degrees instantly
+     *
+     * @param degrees Number of degrees to rotate object
+     */
+    rotateInstantly(degrees) {
+        //
+    }
+
+    /**
+     * Flips object over
+     */
+    flip() {
```

Comment:
To follow the naming scheme, lets change these to `setFlip(faceUp, time)` and `setFlipInstant()`